core-ui/src/components/Extensibility/components/tests/Table.test.js

```diff
@@ -20,6 +20,15 @@ jest.mock('react-i18next', () => ({
   }),
 }));
 
+jest.mock('react-i18next', () => ({
+  useTranslation: () => ({
+    i18n: {
+      exists: () => true,
+    },
+    t: value => value,
+  }),
+}));
+
```

Comment:
What about adding this `exists: () => true` to our global mock in `react-i18-next`?